dora/tests/testcontainers/src/test/java/alluxio/membership/MembershipManagerTest.java

```diff
@@ -266,7 +277,11 @@ public void testFlakyNetwork() throws Exception {
     CommonUtils.waitFor("Workers joined",
         () -> {
           try {
-            return !membershipManager.getLiveMembers().isEmpty();
```

Comment:
i prefer the old API, it's cleaner. We can even add a bit more like `membershipManager.hasLiveMembers()`